lib/widgets/login.dart

```diff
@@ -161,30 +183,61 @@ class _EmailPasswordLoginPageState extends State<EmailPasswordLoginPage> {
   Widget build(BuildContext context) {
     assert(!PerAccountStoreWidget.debugExistsOf(context));
     return Scaffold(
-      appBar: AppBar(title: const Text('Log in')),
+      appBar: AppBar(title: const Text('Log in'),
+        bottom: _inProgress
+          ? const PreferredSize(preferredSize: Size.fromHeight(4),
+              child: LinearProgressIndicator(minHeight: 4))
+          : null),
       body: SafeArea(
         minimum: const EdgeInsets.all(8),
         child: Center(
           child: ConstrainedBox(
             constraints: const BoxConstraints(maxWidth: 400),
             child: Form(
-              child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
-                TextFormField(
-                  key: _emailKey,
-                  keyboardType: TextInputType.emailAddress,
-                  decoration: const InputDecoration(
-                    labelText: 'Email address')),
-                const SizedBox(height: 8),
-                TextFormField(
-                  key: _passwordKey,
-                  obscureText: true,
-                  keyboardType: TextInputType.visiblePassword,
-                  decoration: const InputDecoration(
-                    labelText: 'Password')),
-                const SizedBox(height: 8),
-                ElevatedButton(
-                  onPressed: _submit,
-                  child: const Text('Log in')),
-              ]))))));
+              child: AutofillGroup(
```

Comment:
At least on iOS, this lets me use my password manager to autofill my
    email and password together in a single interaction. Before, it was
    only autofilling the password field.

Interesting! Seems to work for me, on Android, too.